code/game/objects/items/fallout13misc.dm

```diff
@@ -167,48 +167,63 @@
 	icon_state = "ncrflag"
 	item_state = "ncrflag"
 	faction = "NCR"
+	anchored = 1
 
 /obj/item/flag/bos
 	name = "Brotherhood of Steel flag"
 	desc = "A red and black flag with a sword surrounded in gears and wings, in a dazzling gold."
 	icon_state = "bosflag"
 	item_state = "bosflag"
 	faction = FACTION_BROTHERHOOD
+	anchored = 1
 
 /obj/item/flag/legion
 	name = "Legion flag"
 	desc = "A flag with a golden bull, the symbol of Caesar's Legion."
 	icon_state = "legionflag"
 	item_state = "legionflag"
 	faction = FACTION_LEGION
+	anchored = 1
+
+obj/item/flag/bighorn
```

Comment:
/obj/item/flag/bighorn

Reply:
fixed, thought linters were just giving a error because it's linters. ty